diff --git a/contracts/BordaFixed.sol b/contracts/BordaFixed.sol
index d7cd2b9..712e70a 100644
--- a/contracts/BordaFixed.sol
+++ b/contracts/BordaFixed.sol
@@ -89,8 +89,8 @@ contract Borda is IBorda {
 
         _voters[msg.sender].voted = true;
         voteTo(first, 3);
-        voteTo(second, 2);
-        voteTo(third, 1);
+        voteTo(second, 3);
+        voteTo(third, 3);
 
         return true;
     }
