diff --git a/contracts/BordaFixed.sol b/contracts/BordaFixed.sol
index ef88d11..240b344 100644
--- a/contracts/BordaFixed.sol
+++ b/contracts/BordaFixed.sol
@@ -60,7 +60,7 @@ contract Borda is IBorda {
     }
 
     function registerVoter(uint8 age) external override returns (bool) {
-        require (!_voters[msg.sender].registered, "you are already registered");
+        require (!_contenders[msg.sender].registered, "you are already registered");
         _voters[msg.sender] = Voters({age: age, registered: true, voted: false, vote_attempts: 0, black_listed: false});
         return true;
     }
