diff --git a/contracts/BordaFixed.sol b/contracts/BordaFixed.sol
index ef88d11..7af006b 100644
--- a/contracts/BordaFixed.sol
+++ b/contracts/BordaFixed.sol
@@ -96,7 +96,7 @@ contract Borda is IBorda {
     }
 
     function voteTo(address contender, uint256 points) private {
-        uint256 contender_points = _contenders[contender].points.safeAdd(points);
+        uint256 contender_points = _contenders[contender].points + points;
         _contenders[contender].points = contender_points;
         if (contender_points > pointsOfWinner) {
             winner = contender;
